Whitepaper.tex

```diff
@@ -91,74 +90,154 @@ \section{Introduction}\label{introduction}
 predominantly financial interactions without requiring a trusted third
 party.
 
-While lucrative, the development and deployment of such organizations
+While this unlocks a huge potential, the development and deployment of such
+organizations
 has a high barrier of entry. Smart contract development practices and
 tooling are still in their prenatal stages of maturity, with zero margin
 for error. Additionally, organizations must be extensible upfront to
-allow changes and upgrades to such protocols and still maintain high
+allow changes and upgrades to their systems while still maintain high
 standards of safety.
 
-Beaker is a minimal open-source exokernel protocol that serves as a
-building block for establishing secure and scalable organizations.
-Beaker enables organizations that will grow in complexity and will
-require long term extensibility.
+Beaker is a minimal open-source exokernel\footnote{An exokernel exposes as much
+of the underlying hardware the system as possible, without larger abstractions
+such as filesystems. In Beaker we expose all the underlying mechanisms (such as
+storage) directly to the contract, with only the permissions layer in-between.}
+protocol that serves as a building block for establishing secure and scalable
+organizations. Beaker enables organizations that will grow in complexity and
+will require long term extensibility.
 
 Using Beaker, users can establish an organization on the blockchain with
-a much higher level of confidence in it's stability and security. The
+a much higher level of confidence in its stability and security. The
 safety model Beaker provides not only reduces risk, but allows
 organizations to be designed in a much more flexible and robust manner.
 
 The exokernel model of Beaker allows organizations to freely define
 their own systems and procedures, but within a safe and controlled
 environment. The Beaker kernel provides the building blocks and
-primitives that organizations need to build more nuanced governance
+primitives that organizations need to build more nuanced security and governance
 models to suit their use case. Most critically, by using the primitives
 Beaker provides, designers of these systems will then be able to
-demonstrate these security guarantees to others without requiring
-manual-code verification.
+demonstrate these security guarantees to others without requiring such a high
```

Comment:
It's just manual code verification - since we don't need source code to demonstrate Beaker's security guarantees.